config/routes/support.rb

```diff
@@ -14,6 +14,9 @@
     get '/change-course-choice/:application_choice_id' => 'application_forms/courses#edit', as: :application_form_change_course_choice
     post '/change-course-choice/:application_choice_id' => 'application_forms/courses#update'
 
+    get '/editable-until' => 'application_forms/editable_until#edit'
```

Comment:
The `editable-until` naming doesn't feel right as you can't configure `until` as a user, and you can only know the `until` value after you submitted the form.

Reply:
Do you have a suggestion for it? That's the best I could come up with 😂😂😂😂. I promise to name things better in the future 🤞

Reply:
I'm not any better 😆 Saw you named a method `granted_editable_extension`. `editable-extension` sounds better to me than `editable-until`

Reply:
Done!